src/viewer.js

```diff
@@ -3200,14 +3201,15 @@ function onCanvasPress( event ) {
 
     gestureSettings = this.gestureSettingsByDeviceType( event.pointerType );
     if ( gestureSettings.dblClickDragToZoom ){
+        var dblClickTimeThreshold = THIS[ this.hash ].dblClickTimeThreshold;
         var lastClickTime = THIS[ this.hash ].lastClickTime;
         var currClickTime = $.now();
 
         if ( lastClickTime === null) {
             return;
         }
 
-        if ((currClickTime - lastClickTime) < 2000) {
+        if ((currClickTime - lastClickTime) < dblClickTimeThreshold) {
```

Comment:
Sorry I wasn't more clear before... `dblClickTimeThreshold` is a property of `Viewer` so here you just need to use `this.dblClickTimeThreshold`. It already has a default value and it's documented so people can customize it if they want.

Reply:
I have made the change. Please Check and verify on each screen before i resolve this issue